src/Compilers/Core/Portable/CommandLine/SarifV1ErrorLogger.cs

```diff
@@ -44,7 +44,11 @@ public SarifV1ErrorLogger(Stream stream, string toolName, string toolFileVersion
             _writer.Write("version", toolAssemblyVersion.ToString());
             _writer.Write("fileVersion", toolFileVersion);
             _writer.Write("semanticVersion", toolAssemblyVersion.ToString(fieldCount: 3));
-            _writer.Write("language", culture.Name);
+
+            // Emit the 'language' property only if it is a non-empty string to match the SARIF spec.
+            if (culture.Name.Length > 0)
```

Comment:
What about language properties that are all whitespace?

Reply:
I wasn't sure if that is a valid culture name. The docs at https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.name?view=net-7.0#property-value indicate it will be an empty string or in the expected format.

Reply:
All the CultureInfo constructors should throw `CultureNotFoundException` for invalid culture identifiers: https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.-ctor?view=net-7.0